services/debugger/source/eventUploader.go

```diff
@@ -42,12 +43,13 @@ var (
 	configBackendURL    string
 	disableEventUploads bool
 	pkgLogger           logger.Logger
-	eventsCacheMap      debugger.Cache[[]byte]
+	eventsCache         cache.Cache[[]byte]
 )
 
 func Init() {
 	loadConfig()
 	pkgLogger = logger.NewLogger().Child("debugger").Child("source")
+	eventsCache, _ = cache.New[[]byte](cache.BadgerCacheType, pkgLogger)
```

Comment:
@fracasula Is there any nicer way where I can toggle the type instead of hardcoding this? All I can think of is to have a separate function in `config` which returns the CacheType output

Reply:
Not sure I follow. Do you need this to switch from `[]byte` to something else in `Init()`?

Reply:
I want to be able to change `cache.BadgerCacheType` to something else via config